cmd/nvidia-dra-plugin/nvlib.go

```diff
@@ -167,6 +175,8 @@ func (l deviceLib) getGpuInfo(index int, device nvdev.Device) (*GpuInfo, error)
 		brand:                 brand,
 		architecture:          architecture,
 		cudaComputeCapability: cudaComputeCapability,
+		driverVersion:         driverVersion,
+		cudaDriverVersion:     fmt.Sprintf("%v.%v", cudaDriverVersion/1000, (cudaDriverVersion%1000)/10),
```

Comment:
Out of scope for this PR. But do we want to pull this into gonvlib (GetCUDADriverVersionString or something?)

Reply:
Yes, probably.

Reply:
Let#s do it as a follow-up